src/state/ducks/Contacts.ts

```diff
@@ -78,10 +78,8 @@ class Store {
 
         const contact = state.getIn(['contacts', address])
         const { metaData, publicIdentity } = contact
-
```

Comment:
Change request:
In general, if your ticket doesn't affect a file functionally, the file shouldn't be part of your PR. It makes it easier for you and your reviewer!

Reply:
Yes, I have updated it.